pegasus/sites.v3/code.org/views/course_curriculum.haml

```diff
@@ -18,8 +18,8 @@
 - elsif series == "2.0"
   %p
     = I18n.t("dashboard_#{course}")
-    %a{href: "https://studio.code.org/s/#{course}-2021", target: '_blank'}
+    %a{href: "https://studio.code.org/s/#{course}-2022", target: '_blank'}
       = I18n.t :dashboard_lesson_plans
     |
-    %a{href: "https://studio.code.org/s/#{course}-2021/standards", target: '_blank'}
+    %a{href: "https://studio.code.org/s/#{course}-2022/standards", target: '_blank'}
```

Comment:
It looks like this is commented out. Should it just be removed if its commented out?

Reply:
this is not actually commented out, it actually affects what I see here:
![Screen Shot 2022-06-29 at 6 00 17 PM](https://user-images.githubusercontent.com/8001765/176570665-37937504-bbe3-4c5b-b3b5-4d607c0e5f69.png)